index.html

```diff
@@ -10,15 +10,11 @@
         // Initialize framework 
         VSS.init({
             explicitNotifyLoaded: true,
-            usePlatformScripts: true,
-            configureModuleLoader: true
+            usePlatformScripts: true
```

Comment:
do we not need configureModulesLoader?

Reply:
I haven't found any documentation about that option does or did. It didn't seem necessary in my testing.

Reply:
I think this isn't supported anymore, but has stuck around in extension samples. It should be safe to remove.